src/global.d.ts

```diff
@@ -0,0 +1,2 @@
+// TODO(oskar) - prepare type interfaces that we use from this package.
+declare module 'ethereumjs-tx';
```

Comment:
@opaolini can you actually flesh this out in this PR? There are plenty of examples in `@0x/typescript-typings`

Reply:
```
src/global.d.ts:4:16 - error TS2665: Invalid module name in augmentation. Module 'ethereumjs-tx' resolves to an untyped module at '~/projects/0x/0x-api/node_modules/ethereumjs-tx/es5/index.js', which cannot be augmented.
```

And unfortunately I am not entirely sure how to fix this.